background/KF.ts

```diff
@@ -3,6 +3,7 @@
 /// <reference path="../common/Logger.ts" />
 /// <reference path="commands.ts" />
 /// <reference path="utils.ts" />
+/// <reference path="jsonrpcClient.ts" />
```

Comment:
The fact you had to include this suggests to me that we're using some different IDEs. VS Code works just fine without this because it understands tsconfig.json files and treats all ts files in the background directory as part of the project and hence we don't normally need to explicitly reference every file.